release/apparmor.go

```diff
@@ -83,43 +84,59 @@ var (
 	// appArmorParserError contains an error, if any, encountered when
 	// discovering available parser features.
 	appArmorParserError error
+
+	appArmorProbeParserOnce sync.Once
+	appArmorProbeKernelOnce sync.Once
+	appArmorAssessOnce      sync.Once
+
+	appArmorProbeKernel = probeAppArmorKernelFeaturesOnce
+	appArmorProbeParser = probeAppArmorParserFeaturesOnce
+	appArmorAssess      = assessAppArmorOnce
 )
 
+func assessAppArmorOnce() {
+	appArmorAssessOnce.Do(assessAppArmor)
```

Comment:
were the actual functions like assesAppArmor etc tested?

Reply:
Yes, they are tested in a couple of places: 
- https://github.com/snapcore/snapd/pull/6306/files#diff-ecc06490ecae634064182423ba57f533R197
- https://github.com/snapcore/snapd/pull/6306/files#diff-ecc06490ecae634064182423ba57f533R163
- https://github.com/snapcore/snapd/pull/6306/files#diff-ecc06490ecae634064182423ba57f533R138